data_profiller/utils/data_collection.py

```diff
@@ -93,31 +93,57 @@ def get_init_dictionary(headings):
     return column_datatype_line_no
 
 
+def isFloat(val):
+    '''
+    Return true if input parameter is of type float.
+    '''
+    try:
+        float(val)
+
+        return True
+    except:
+
+        return False
+
+
+def isInt(val):
+    '''
+    Return true if input parameter is of type integer.
+    '''
+    if '.' in str(val):
+
+        return False
+
+    try:
+
+        return isinstance(int(float(val)), int)
+
+    except:
+
+        return False
+
+
 def get_datatype(cell_value):
-    ''' Return datatype of input parameter '''
+    '''Return datatype of input parameter in string formate'''
 
     if cell_value == 'NULL':
         dtype_cell = None
+    elif is_date(cell_value):
+        dtype_cell = 'date'
```

Comment:
Why are you assigning these values if all you do is return them later? You can return right away.
```suggestion
        return 'date'
```